protecode/util.py

```diff
@@ -70,70 +66,11 @@ def upload_images(
         ignore_if_triaged=ignore_if_triaged,
     )
 
-    if upload_registry_prefix:
-        # XXX this is an ugly hack / workaround (see below)
-        _download_images(
-            component_descriptor=product_descriptor,
-            upload_registry_prefix=upload_registry_prefix,
-            image_reference_filter=image_reference_filter,
-        )
-
     _license_report = license_report(upload_results=results)
 
     return (relevant_results, _license_report)
 
 
-def _download_images(
```

Comment:
clarify with @ThormaehlenFred whether this functionality is still required. if so, mv equivalent functionality to upload-images trait